WebPortal/Controllers/ApiController.cs

```diff
@@ -184,7 +184,8 @@ public async Task<IActionResult> GetFieldChoices()
             return Ok(fieldChoices);
         }
 
-        private bool HasExpired(Datahub_ProjectApiUser apiUser) => apiUser.Expiration_DT.HasValue && apiUser.Expiration_DT.Value > DateTime.UtcNow;
+        private bool IsDisabledOrExpired(Datahub_ProjectApiUser apiUser) 
+            => !apiUser.Enabled && apiUser.Expiration_DT.HasValue && apiUser.Expiration_DT.Value > DateTime.UtcNow;
```

Comment:
Should this be !enabled || (expiration date check)?

Reply:
Good catch!